styles/prosilver/template/posting_body.html

```diff
@@ -1,91 +1,91 @@
-<!-- INCLUDE overall_header.html -->
+{% INCLUDE 'overall_header.html' %}
 
 {% if SHOW_PREVIEW %}
 	{% INCLUDECSS '@kinerity_knowledgebase/knowledgebase_controller.css' %}
 	{% include 'preview_article.html' %}
 {% endif %}
-<form id="postform" method="post" action="{S_POST_ACTION}"{S_FORM_ENCTYPE}>
+<form id="postform" method="post" action="{{ S_POST_ACTION }}"{{ S_FORM_ENCTYPE }}>
 
 <div class="panel" id="postingbox">
 	<div class="inner">
 
-	<h3>{L_KB_POST_A}</h3>
+	<h3>{{ lang('KB_POST_A') }}</h3>
```

Comment:
Das muss {{ L_KB_POST_A }} lauten, weil der Variablen die entsprechenden Werte schon mit z.B. 
$page_title = $this->lang->lang('KB_POST_ARTICLE');
zugewiesen werden

Reply:
Behoben, ist ja eine Template-Variable, also ohne L_ umbenannt.